### DISCUSSION TITLE
Popup box when merge branch

### DISCUSSION THREAD

**Discussion by divyeshgohil85** [community/community#72183] [Repositories]
### Select Topic Area

Question

### Body

As per experience and dally practice i am used to merge branch in github but sometime i make a mistake like merge wrong branch
why github should not introduce popup box when we merge a branch
Like are you sure want to merge this branch to this branch.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @divyeshgohil85! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you so that your feedback can get visibility from the appropriate teams here. Good luck!
